panels/UserPanel.php

```diff
@@ -198,33 +198,37 @@ public function save()
         $identity = Yii::$app->user->identity;
 
         if (!isset($identity)) {
-            return ;
+            return;
         }
 
-        $authManager = Yii::$app->getAuthManager();
-
         $rolesProvider = null;
         $permissionsProvider = null;
 
-        if ($authManager instanceof yii\rbac\ManagerInterface) {
-            $roles = ArrayHelper::toArray($authManager->getRolesByUser(Yii::$app->getUser()->id));
-            foreach ($roles as &$role) {
-                $role['data'] = $this->dataToString($role['data']);
-            }
-            unset($role);
-            $rolesProvider = new ArrayDataProvider([
-                'allModels' => $roles,
-            ]);
-
-            $permissions = ArrayHelper::toArray($authManager->getPermissionsByUser(Yii::$app->getUser()->id));
-            foreach ($permissions as &$permission) {
-                $permission['data'] = $this->dataToString($permission['data']);
-            }
-            unset($permission);
+        try {
```

Comment:
It can't in this case cause misconfiguration could be anything. In the case of the issue reported it was trying to set non-existing property of the PHP-backed auth manager as PHP manager doesn't support caching.